crates/extensions/tedge_mqtt_bridge/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "tedge_mqtt_bridge"
+description = "thin-edge extension adding a bridge MQTT connection to the desired cloud provider"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+
+[features]
+# No features on by default
+default = []
+
+[dependencies]
+async-trait = { workspace = true }
+c8y_api = { workspace = true }
```

Comment:
It would be really better not to depend on `c8y_api`.

Some dependencies are not used (e.g. `serde_json` and `tedge_utils`). Maybe some more.